mjsdevs/vitae-web
# Feat/login right section

## What I did:
- Created component LoginRightSection
- Added "Fork me on GitHub" ribbon
- Added Big logo to component
- Add purple background-color to component
- Added component to Login Page

## How to test:
- Run `yarn` to install the dependencies
- Run `yarn test` to run the test suits
- Run `yarn start` to start the project
- Then go to http://localhost:3000, there you should see the Login page
- Try clicking on the ribbon, you should be redirected to this repository

resolves #2
